SPEC.md

```diff
@@ -0,0 +1,334 @@
+# Nexus RPC HTTP Specification
+
+## Overview
+
+The Nexus protocol, as specified below, is a synchronous RPC protocol. Arbitrary length operations are modelled on top
```

Comment:
I know that the term has been in use for a while, but I personally vote for "arbitrary duration operations" rather than "length". It's time that's being referred to, so duration is more correct. (If an acronym starts getting thrown around, then it's interesting that "ADO" brings "async do" to mind -- possibly fortuitous?)

Reply:
@rylandg please consider

Reply:
ADO makes me think of my old MS coding days and Active Data Objects (but I don't disagree...there's no TLA that isn't already taken somehow)

Reply:
You need something to differentiate a regular operation and an operation that returns a "promise" if you will. Async operation is fine with me. I do not believe the HTTP API proposal is where to hash out naming (also, consult with Ryland here).